app/actions/v3/service_plan_visibility_update.rb

```diff
@@ -69,10 +74,14 @@ def unprocessable!(message)
         raise UnprocessableRequest.new(message)
       end
 
-      def space?(service_plan)
+      def visibility_type_space?(service_plan)
         service_plan.visibility_type == VCAP::CloudController::ServicePlanVisibilityTypes::SPACE
       end
 
+      def visibility_type_org?(service_plan)
+        service_plan.visibility_type == VCAP::CloudController::ServicePlanVisibilityTypes::ORGANIZATION
+      end
+
       def org?(type)
         type == VCAP::CloudController::ServicePlanVisibilityTypes::ORGANIZATION
       end
```

Comment:
Can we combine this into a single method and use the same pattern for `visibility_type_space`?